### DISCUSSION TITLE
library/i2c

### DISCUSSION THREAD

**Discussion by giscus** [py32duino/document#6] [comment]
# library/i2c

关于 I2C（Inter-Integrated Circuit）/TWI（Two-wire Interface）是一种广泛使用的串行通信，用于短距离连接设备。这是用于连接传感器、EEPROM、RTC、ADC、DAC、显示器、OLED 以及许多其他设备和微控制器的最常见外设之一。 这种串行通信被视为低速总线，多个设备可以连接在同一条两线总线上，每个设备都...

https://arduino.py32.org/library/i2c.html

**Comment by ytangy82** [comment]
setSDA() 传的参数，怎么和PinMap_I2C_SDA 这个数组对应，没有看明白。最好有个示例。

**Comment by ShuangGeng** [comment]
从机模式不能工作
